### PR TITLE
Disable PrivateNetworkAccessForNavigations* via a kill switch

### PR DESCRIPTION
**Pull request by atuchin-m**
Resolves https://github.com/brave/brave-variations/issues/970
PR to main: https://github.com/brave/brave-variations/pull/969

The chromium bug:  https://issues.chromium.org/issues/330594669

### CODE REVIEW

**Review by iefremov — DISMISSED** [review]

**Review by kjozwiak — APPROVED** [review]
Uplift into `production` approved after deliberating with @brave/uplift-approvers.

**Pull request merged**
